### PR TITLE
Make video draggable and add animation

### PR DESCRIPTION
**Pull request by alsile**
This PR:
- improves style of previous code (putting common code into functions, removing unused code, removing debugging statements etc.)
- adds markers wherever users search/click
- makes videos draggable and resizable by dragging the white border and editing the edges of the video
- creates minimize and maximize animation from where the marker is when videos are played
- change enter key to effectively press the video and location search icons
- added alerts for errors that cause disfunctionality
- fix bug where location radius was too large if zoom was far enough
- i cant think of anything else but it's possible that this changes other stuff too, but as far as my tests go it doesn't break anything

I will add a demo tomorrow when our quota resets.

### CODE REVIEW

**Review by kinsaurralde — APPROVED** [review]

**Comment by kinsaurralde** [comment]
I noticed dragging the video window to the right and bottom expands the screen (top and left are fine).

Screenshot 2020-08-18 at 8 49 57 AM - Display 1

**Comment by alsile** [comment]
> I noticed dragging the video window to the right and bottom expands the screen (top and left are fine).
> 
> Screenshot 2020-08-18 at 8 49 57 AM - Display 1

Thanks Kyle, just fixed this

**Review by panerorenn9541 — APPROVED** [review]

**Comment by alsile** [comment]
test (2)

**Review by v01dstar — APPROVED** [review]
LGTM

File: src/main/webapp/get-videos.js [hunk 1]
```diff
@@ -10,39 +10,42 @@ function searchForVideos(map, searched) {
         .then(
             function() {
               executeSearch(map, searched);
-              console.log('load was successful');
+              // console.log('load was successful');
```
Comment (huzcn) [inline src/main/webapp/get-videos.js]: Remove this line if it's no longer needed.

**Review by huzcn — APPROVED** [review]
Nice work!

**Pull request merged**